### ISSUE TITLE
Client information shown with ipv6 enabled

### ISSUE THREAD

**Issue by kkfuzi** [AdguardTeam/AdGuardHome#1987]
Please answer the following questions for yourself before submitting an issue. 
- [*] I am running the latest version
- [*] I checked the documentation and found no answer
- [*] I checked to make sure that this issue has not already been filed

### Problem Description
Hi Team,
I have a question about client information shown with ipv6 enabled.

I'm having OpenWrt installed on my router and Adguard Home running as a service on it.
IPv6 is enabled in my network while DHCP is set up on OpenWrt system.

I'm using ADH mode "replace dnsmasq with port 53" in OpenWrt configuration portal and I can see the client information in the AGH dashboard. But some of the client addresses are the dynamic internal IPv6 address (I'm setting IPv6 ULA prefix as fd29:4738:32a7::/48 on OpenWrt). Since the IPv6 will change randomly, I'm not able to create the client based on the IPv6 address and cannot use a customized filtering rule for some of the clients (such as my iPad).

Can you help to see what kind of workaround in this case I can use so I'm able to bind the client with IP while IPv6 still works in my network?

Thanks.

![Screen Shot 2020-08-03 at 13 05 23_meitu_1](https://user-images.githubusercontent.com/19731165/89149046-0ee23700-d58e-11ea-8f3f-edf1104d742a.jpg)

**Comment by szolin** [comment]
Can you use static IPv6 address (in OpenWrt DHCP configuration) for each device?

**Comment by kkfuzi** [comment]
> Can you use static IPv6 address (in OpenWrt DHCP configuration) for each device?

That's something I've tried already from openwrt side. I think there is no Native Static IPv6 option in LuCI-Interfaces dropdown. only having an "IPv6 suffix" option after the native static IPv4 setup.

For example I've set my Macbook with static IPv4 address and an IPv6 suffix "a110", what shows on my mac is the below
The address starting with 240e.xxx are from my ISP, public IPv6 
The address starting with fd29.xxx are internal communication IPv6

seems that the suffix are set to the public address but on AGH it is using the internal private one.

Being troubled by this issue for a long time..

![Screen Shot 2020-08-03 at 18 10 02_meitu_1](https://user-images.githubusercontent.com/19731165/89172944-07cf1f00-d5b6-11ea-81c7-fa6072f9f64d.jpg)

**Comment by szolin** [comment]
I only can suggest to wait for the future release of AGH where we add DHCPv6 server capability.  You'll be able to assign a static DHCPv6 addresses and resolve your problem.
#779

**Comment by kkfuzi** [comment]
So let me rephrase my question to think differently. If a client has both ipv4 and ipv6 addresses, how can I let the client to use ipv4 address with higher prority than ipv6 when doing DNS lookup? This might fix the problem I have currently...

**Comment by szolin** [comment]
Only if you disable AAAA requests in AGH or disable IPv6 completely.

**Comment by kkfuzi** [comment]
tried, disable AAAA not work, as the v6 address chosen is from the client side, not affected by this change.
disable v6 completely, hmmm, i'm afraid i still need to use v6 for some client (NAS and pt download..)

anyway, thanks for the feedback, i believe there's not too much related to AGH for now.

**Comment by netitgo** [comment]
same demand for the IPV4 & IPV6 options, so that we could choose what we want shows in the Client field.
